kubediff

```diff
@@ -1,50 +1,97 @@
 #!/usr/bin/env python
+# -*- coding: utf-8 -*-
 
-import optparse
+from __future__ import (division, absolute_import, print_function,
+                        unicode_literals)
+
+import argparse
 import sys
 
 from kubedifflib import (
-  check_files,
-  JSONPrinter,
-  QuietTextPrinter,
+    check_files,
+    JSONPrinter,
+    QuietTextPrinter,
 )
 
 
+class ParseArgs():
+    def __init__(self):
+
+        # Parse arguments passed at cli
+        self.parse_arguments()
+
+    def parse_arguments(self):
+
+        description = '''
+     _          _             _  _   __   __
+    | |__ _  _ | |__  ___  __| |(_) / _| / _|
+    | / /| || || '_ \/ -_)/ _` || ||  _||  _|
+    |_\_\ \_,_||_.__/\___|\__,_||_||_|  |_|
+
```

Comment:
```cli
kubediff:25:112: W1401: Anomalous backslash in string: '\/'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
kubediff:25:144: W1401: Anomalous backslash in string: '\_'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
kubediff:25:146: W1401: Anomalous backslash in string: '\ '. String constant might be missing an r prefix. (anomalous-backslash-in-string)
kubediff:25:148: W1401: Anomalous backslash in string: '\_'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
kubediff:25:159: W1401: Anomalous backslash in string: '\_'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
kubediff:25:164: W1401: Anomalous backslash in string: '\_'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
```

Looks like `pylint` is a bit unhappy about this...? :grimacing:

Reply:
Let's keep it simple for now 👯

Reply:
Escaped and Fixed. Thanks @dholbach